activerecord/lib/active_record/relation/query_methods.rb

```diff
@@ -950,6 +924,17 @@ def arel # :nodoc:
       @arel ||= build_arel
     end
 
+    # Returns a relation value with a given name
+    def get_value(name) # :nodoc:
+      @values[name] || default_value_for(name)
+    end
+
+    # Sets the relation value with the given name
+    def set_value(name, value) #:nodoc:
```

Comment:
I am a little curious about `#:nodoc:`. `get_value` is `# :nodoc:` (including a space).